test/known_issues/test-vm-timeout-escape-queuemicrotask.js

```diff
@@ -8,6 +8,12 @@ const common = require('../common');
 const assert = require('assert');
 const vm = require('vm');
 
+common.skip(
+  'Test is inherently flaky and will not be fixed without major changes to V8' +
+  ' engine which is unlikely to happen in the foreseeable future.' +
```

Comment:
It’s not so much that the test is inherently flaky – it’s that the *feature* that it asks for won’t be implemented.

Reply:
Right, this sounds like the issue is in the flakiness and not the feature itself.
How does this sound:
`'Due to the implementation details of the V8 engine and Node.js, this issue will not be fixed without major changes that are unlikely to happen in the foreseeable future. The test itself is flaky and due to the unactionable fashion of the issue is skipped and left only as an example.'`

Reply:
Maybe this?

```javascript
// This issue is unfixable with the current implementations of V8 and Node.js.
// Changes to make this fixable are unlikely in the foreseeable future. The test
// itself is flaky and skipped. It serves as a demonstration of the issue only.
```

Reply:
Maybe `unlikely to happen in the ..`, otherwise LGTM. 
@addaleax wdyt?

Reply:
@lundibundi Yeah, I’m good with that :)